helm/charts/hydra/templates/configmap.yaml

```diff
@@ -5,6 +5,10 @@ metadata:
   namespace: {{ .Release.Namespace }}
   labels:
 {{ include "hydra.labels" . | indent 4 }}
+  annotations:
+    helm.sh/hook-weight: "0"
+    helm.sh/hook: "pre-install, pre-upgrade"
+    helm.sh/hook-delete-policy: "before-hook-creation"
```

Comment:
this delete policy hook may be risky: if upgrade fails for some reason, helm will delete the CM already leaving hydra without config.  If someone is having rollback this could help him but if someone is not using rollback may run into trouble.